citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -104,7 +134,156 @@
 	    $('.note-info').each(function() {
 	    	$(this).text(truncateNote($(this).attr('data-note-text')));
 	    })
+	    
+	    $("#closeReferenceModalBtn").click(function() {
+	    	$("#referenceModal").modal('hide');
+	    });	    
+
+	    $("#addReferenceButton").click(function() {
+	        $("#addReferenceSpinner").show();
+	    	document.getElementById('addReferenceSpinner').style.display = 'inline-block';
+
+		var zoteroGroupId = $("#addReferences").data('zotero-group-id');
+	        var itemId = $("#addReferences").data('item-id');
+	        var reference =  $("#itemReference").text().trim();
+	        var referenceCitationKey =  $("#itemReference").data('citation-key');
+	        addItemReference(zoteroGroupId, itemId, referenceCitationKey, reference);
+	    });
+	    
+	    $("#searchReference").click(function() {
+	        $("#selectReferenceModal").show()
+	        $("#searchReferenceSpinner").show();
+	        $("#addReferenceButton").prop('disabled', true);
+	        $("#itemReference").val('');
+	        var searchTerm = $("#citationReference").val();
+	        var zoteroGroupId = $("#addReferences").data('zotero-group-id');
+	        getReferences(zoteroGroupId, searchTerm);
+	        $("#searchReferenceSpinner").hide();
+	    });
+	    
+	    $("#closeReferenceSearchResult").click(function() {
+	        $("#selectReferenceModal").modal('hide');
+	        $("#searchReferenceSpinner").hide();
+	        $("#addReferenceSpinner").hide();
+	    });
+	    
+	    $("#addReferences").click(function() {
+	        $("#citationReference").val('');
+		$("#itemReference")[0].innerHTML = '';
+	    });
+	    
+	    $("#referenceModalBody").on('keypress', function(event) {
+	        if (event.key === 'Enter') {
+	            event.preventDefault(); 
+	            document.getElementById('searchReference').click();
+	        }
+	    });
+	    
 	});
+
+	function getReferences(zoteroGroupId, searchTerm) {
+	    var url = [[@{|/auth/group/|}]] + zoteroGroupId + '/citations/search' + '?searchTerm=' + searchTerm;
+	    $.ajax({
+	        dataType: "json",
+	        type: 'GET',
+	        url: url,
+	        async: false,
+	        success: function(data) {
+	            var results = data["results"];
+	            if(results.length==0) {
+	            	$("#citationSearchTable").empty();
+	            	var content = '<div class="alert warning">'
+	            	content += '<strong>Warning!</strong>'
+	            	content += ' No citations found for the search query. Try other search query.'
```

Comment:
The message should just be: "No citations were found for the entered search term."
(No "warning" no "try other search query")